source/developer-guides/payload.rst

```diff
@@ -9,14 +9,19 @@ Payloads
 ^^^^^^^^^^^^^^^^^^^
 
 Os Loader (Built-in)
-  A versatile linux loader implementation that boots Linux, hypervsior, Android, or ELF executable, PE executables. It supports MultiBoot specification
+  A versatile linux loader implementation that boots Linux, hypervsior, Android, and ELF or PE executables. It also supports launching OSes compliant with the MultiBoot specification.
 
 Firmware Update Payload (Built-in)
   A special purpose payload to update full boot flash in a secure and fault-tolerant flow.
 
 UEFI Payload (External)
   A EDK II based payload implementation to boot Windows. It provides secure boot, SMM, and UEFI runtime services.
 
+Os Loader (when built as an FV)
+  Provides the same functionality as the non-FV build of OS Loader but permits the launching of a pre-OS checker binary that will hand-off control to an OS after execution.
+

```

Comment:
Pre-OS checker is one use case. Prefer to use more generic description. Something like "When built as an FV, OsLoader can support launching another payload before transferring control to the OS".